model_cards/aliosm/ai-soco-c++-roberta-small-clas/README.md

```diff
@@ -0,0 +1,56 @@
+---
+language: "c++"
+tags:
+- exbert
+- authorship-identification
+- fire2020
+- pan2020
+- ai-soco
+- classification
+license: "mit"
+datasets:
+- ai-soco
+metrics:
+- accuracy
```

Comment:
do we support those (accuracy, perplexity, etc.) in metrics @lhoestq @thomwolf ?

Reply:
no but we can add them at some point